pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -81,6 +82,7 @@ let
         knm            = versionAtLeast ccVersion "7.0";
         # AMD
         znver1         = versionAtLeast ccVersion "4.0";
+        znver2         = versionAtLeast ccVersion "8.0";
```

Comment:
I am not sure when `znver2` support has been added no `gcc` and `clang`, so these version numbers are not final yet